roachtest: lib/pq failed

roachtest.lib/pq [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4820270&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4820270&tab=artifacts#/lib/pq) on master @ [9bff1de7115984957978416d829feb582eb2664e](https://github.com/cockroachdb/cockroach/commits/9bff1de7115984957978416d829feb582eb2664e):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/lib/pq/run_1
	orm_helpers.go:190,orm_helpers.go:116,java_helpers.go:220,libpq.go:135,test_runner.go:875: 
		Tests run on Cockroach v22.1.0-alpha.4-771-g9bff1de711
		Tests run against lib/pq v1.10.4
		217 Total Tests Run
		179 tests passed
		38 tests failed
		7 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		6 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: pq.TestStmtExecContext/context.WithTimeout_exceeded - unknown (unexpected)
		--- FAIL: pq.TestStmtExecContext - unknown (unexpected)
		--- FAIL: pq.TestStmtQueryContext/context.WithTimeout_exceeded - unknown (unexpected)
		--- FAIL: pq.TestIssue1046 - unknown (unexpected)
		--- FAIL: pq.TestIssue1062 - unknown (unexpected)
		--- FAIL: pq.TestStmtQueryContext - unknown (unexpected)
		For a full summary look at the lib/pq artifacts 
		
		An updated blocklist (libPQBlocklist22_1) is available in the artifacts' lib/pq log
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #79569 roachtest: lib/pq failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1 release-blocker]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*lib/pq.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-14910

--- a/pkg/cmd/roachtest/tests/libpq.go
+++ b/pkg/cmd/roachtest/tests/libpq.go
@@ -25,7 +25,7 @@ import (
 )
 
 var libPQReleaseTagRegex = regexp.MustCompile(`^v(?P<major>\d+)\.(?P<minor>\d+)\.(?P<point>\d+)$`)
-var libPQSupportedTag = "v1.10.4"
+var libPQSupportedTag = "v1.10.5"
 
 func registerLibPQ(r registry.Registry) {
 	runLibPQ := func(ctx context.Context, t test.Test, c cluster.Cluster) {

--- a/pkg/cmd/roachtest/tests/libpq_blocklist.go
+++ b/pkg/cmd/roachtest/tests/libpq_blocklist.go
@@ -17,7 +17,37 @@ var libPQBlocklists = blocklistsForVersion{
 	{"v22.1", "libPQBlocklist22_1", libPQBlocklist22_1, "libPQIgnorelist22_1", libPQIgnorelist22_1},
 }
 
-var libPQBlocklist22_1 = libPQBlocklist21_2
+var libPQBlocklist22_1 = blocklist{
+	"pq.ExampleConnectorWithNoticeHandler":           "unknown",
+	"pq.TestBinaryByteSliceToInt":                    "41547",
+	"pq.TestBinaryByteSlicetoUUID":                   "41547",
+	"pq.TestConnListen":                              "41522",
+	"pq.TestConnUnlisten":                            "41522",
+	"pq.TestConnUnlistenAll":                         "41522",
+	"pq.TestConnectorWithNoticeHandler_Simple":       "unknown",
+	"pq.TestConnectorWithNotificationHandler_Simple": "unknown",
+	"pq.TestCopyFromError":                           "5807",
+	"pq.TestCopyInRaiseStmtTrigger":                  "5807",
+	"pq.TestCopyInTypes":                             "5807",
+	"pq.TestCopyRespLoopConnectionError":             "5807",
+	"pq.TestEncodeAndParseTs":                        "41563",
+	"pq.TestInfinityTimestamp":                       "41564",
+	"pq.TestIssue186":                                "41558",
+	"pq.TestIssue196":                                "41689",
+	"pq.TestIssue282":                                "12137",
+	"pq.TestListenerFailedQuery":                     "41522",
+	"pq.TestListenerListen":                          "41522",
+	"pq.TestListenerReconnect":                       "41522",
+	"pq.TestListenerUnlisten":                        "41522",
+	"pq.TestListenerUnlistenAll":                     "41522",
+	"pq.TestNotifyExtra":                             "41522",
+	"pq.TestPing":                                    "35897",
+	"pq.TestQueryRowBugWorkaround":                   "5807",
+	"pq.TestReconnect":                               "35897",
+	"pq.TestRowsColumnTypes":                         "41688",
+	"pq.TestRuntimeParameters":                       "12137",
+	"pq.TestStringWithNul":                           "26366",
+}
 
 var libPQBlocklist21_2 = blocklist{
 	"pq.ExampleConnectorWithNoticeHandler":           "unknown",

